src/main/java/propertyFiles/MimeTypeDetectionByFileExtension.properties

```diff
@@ -3,6 +3,7 @@ ado=application/x-stata-ado
 dbf=application/dbf
 dcm=application/dicom
 docx=application/vnd.openxmlformats-officedocument.wordprocessingml.document
+eln=application/zip
```

Comment:
I saw "Given that the file is a zip, I didn't edit MimeTypeDisplay or MimeTypeFacets" in the PR but it would be nice (some day) to have a mime type for .eln. I couldn't find an issue for this, so I created one:

https://github.com/TheELNConsortium/TheELNFileFormat/issues/22

With a mime type in place, in Dataverse we can show whatever we want such as "ELN file" or "Electronic Lab Notebooks" rather than whatever we show for zip files.

I'd say we can go ahead and merge this in the spirit of incremental progress, but I'd feel remiss if I didn't at least mention it.

Reply:
:+1: